### PR TITLE
Update README_PODMAN.md

### LINKED ISSUE #37: Please, correct README_PODMAN.md
Example: container name resolution
sudo podman run -dt --name web --network **foobar** quay.io/libpod/alpine_nginx:latest 5139d65d22135e9ecab511559d863754550894a32285befd94dab231017048c2
Note: we use the --network foobar here. Also, in this test image, the nginx server will respond with podman rulez on an http request.
sudo podman run -it --name client --network **cni-podman1** quay.io/libpod/alpine_nginx:latest curl http://web/ podman rulez

This example suggests that it is possible to enable name resolution between containers in different networks. It is against a statement at the beginning of the document: The containers will only be able to resolve each other if they are on the same CNI network.

### PR DESCRIPTION
**Pull request by omarc1492**
fixes #37

### CODE REVIEW

**Comment by baude** [comment]
you need to sign your commit.  git commit --amend -s

**Comment by omarc1492** [comment]
> you need to sign your commit. git commit --amend -s

I did it through github.com, commit is automatically signed correct?

**Comment by baude** [comment]
@omarc1492 apparently not.

**Comment by omarc1492** [comment]
@baude 
This commit was created on GitHub.com and signed with a verified signature using GitHub’s key.
GPG key ID: 4AEE18F83AFDEB23
Learn about signing commits

File: README_PODMAN.md [hunk 1]
```diff
@@ -23,26 +23,34 @@ should already exist.
...
-The following example [configuration file](example/cni-podman1.conflist) shows a usable example for Podman.
 
 ## Example: container name resolution
+**Note**: we use the --network foobar here. Also, in this test image, the nginx server will respond with
+*podman rulez* on an http request.
 
-1. sudo podman run -dt --name web --network foobar quay.io/libpod/alpine_nginx:latest
-    5139d65d22135e9ecab511559d863754550894a32285befd94dab231017048c2
+```console
+foo@bar:~$ sudo podman run -dt --name web --network foobar quay.io/libpod/alpine_nginx:latest
```
Comment (rhatdan) [inline README_PODMAN.md:48]: Please do not add `foo@bar:~$`, this makes the line difficult to cut and paste.

File: README_PODMAN.md [hunk 2]
```diff
@@ -23,26 +23,34 @@ should already exist.
...
+5139d65d22135e9ecab511559d863754550894a32285befd94dab231017048c2
 
-    Note: we use the --network foobar here. Also, in this test image, the nginx server will respond with
-    *podman rulez* on an http request.
-2. sudo podman run -it --name client --network cni-podman1 quay.io/libpod/alpine_nginx:latest curl http://web/
+foo@bar:~$ sudo podman run -it --name client --network foobar quay.io/libpod/alpine_nginx:latest curl http://web/
```
Comment (rhatdan) [inline README_PODMAN.md:51]: Remove foo@bar:~$

**Comment by edsantiago** [comment]
Thank you for jumping in to help! Since you have to update your commit anyway, could I ask you to include a more descriptive commit message? It is traditional (and helpful to future maintainers) to include the "why" -- the reason for this change. A little effort now may save someone a lot of effort in the future. Thanks again.

**Comment by baude** [comment]
it looks like you now have two unsigned commits.  your best best is to squash them and then git commit --amend -s

**Comment by omarc1492** [comment]
@edsantiago Thank you for the suggestion, I've added some comments.

@baude commit's are verified now.

File: README_PODMAN.md (hunk 1 above)
Comment (omarc1492) [reply README_PODMAN.md]: done

File: README_PODMAN.md (hunk 2 above)
Comment (omarc1492) [reply README_PODMAN.md]: done

**Comment by ScottRochford** [comment]
Might be worth fixing the broken link to "The following example configuration file shows a usable example for Podman." while working on this?  I can't figure out where it's really supposed to link to...

**Comment by omarc1492** [comment]
> Might be worth fixing the broken link to "The following example configuration file shows a usable example for Podman." while working on this? I can't figure out where it's really supposed to link to...

The PR I submitted removes the broken link and adds an example.

**Comment by rhatdan** [comment]
@omarc1492 Needs a rebase?

**Comment by voigt** [comment]
Please do users a favor and merge this PR. It is fixing a serious documentation issue, as the current version is plainly wrong. Would have saved me a couple of minutes finding this PR earlier ;)

**Comment by rhatdan** [comment]
Replaced by #48.

**Pull request closed without merging**